conntrack_test.go

```diff
@@ -311,7 +311,7 @@ func TestConntrackTableDelete(t *testing.T) {
 
 	// Flush entries of groupB
 	var deleted uint
-	if deleted, err = h.ConntrackDeleteFilter(ConntrackTable, unix.AF_INET, filter); err != nil {
+	if deleted, err = h.ConntrackDeleteFilters(ConntrackTable, unix.AF_INET, filter); err != nil {
```

Comment:
you may better add a new test or expand this to delete multiple flows with two different filters to exercise the new functionality

Reply:
I'm also considering passing the errors back to the caller. Currently, `Execute()` can return an error, but it is not being captured. 
Also returning the error will be difficult in case of multiple filters, we would have to decide if we want to execute immediately in case of error, or try out all the filters and then return the error at the end.